app/app/api/api_v1/endpoints/divisions.py

```diff
@@ -134,9 +134,10 @@ def add_division_students_by_id(
                     if user.school_id == division.course.term.year.school_id:
                         if student := crud.student.get(db, id=user_id):
                             if student.term_id == division.course.term_id:
-                                division.students.append({"student": student, "batch_number": batch_number + 1})
-                                batch_number = (batch_number + 1) % division.number_of_batches
-                                response["success"].append(student.user_id)
+                                if student not in division.students:
+                                    division.students.append({"student": student, "batch_number": batch_number + 1})
+                                    batch_number = (batch_number + 1) % division.number_of_batches
+                                    response["success"].append(student.user_id)
```

Comment:
Add to `errors` here maybe? "already in division"

Reply:
I asked aniket what he wanted, he said he wanted duplicates to just be ignored for now just as they are in terms.

Reply:
Asked aniket if he wanted me to return an error or just ignore duplicates, he went with the latter option.

Otherwise I could send errors in response both here as well as in terms for duplicate students.

Reply:
IMO that's better, just to stay consistent (i.e. log different types of errors)
Atleast the user can be informed that those students were already in, rather than just assume they weren't and they just got added